platform/qt/README.md

```diff
@@ -0,0 +1,68 @@
+# Mapbox Qt SDK
+
+[![Travis](https://travis-ci.org/mapbox/mapbox-gl-native.svg?branch=master)](https://travis-ci.org/mapbox/mapbox-gl-native/builds)
+
+Provides [Qt](http://www.qt.io/) example applications and APIs via `QMapboxGL`
+and `QQuickMapboxGL`:
+
+**QMapboxGL** - [QObject](http://doc.qt.io/qt-5/qobject.html)-based, implements
+a C++03x-conformant API that has
+been tested from Qt 4.7 onwards (Travis CI currently builds it using both Qt 4
+and Qt 5).
+
+**QQuickMapboxGL** -
+[QQuickFrambebufferObject](http://doc.qt.io/qt-5/qquickframebufferobject.html)-based,
+implements a Qt Quick
+(QML) item that can be added to a scene. Because `QQuickFramebufferObject` has
```

Comment:
We should say "Qt Quick Item" and not Qt Quick (QML) Item. 

"can be added to a scene" is redundant because very Qt Quick Item can be added to a scene.